### PR TITLE
Add dilution counts to pools

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-2043

### CODE REVIEW

**Comment by callunity** [comment]
Would you consider dropping the "Created Date" column since we have a "Last Modified" column?

File: miso-web/src/main/webapp/scripts/list_partition.js [hunk 1]
```diff
@@ -279,6 +279,22 @@ ListTarget.partition = {
...
+            if (data) {
+              return data.dilutionCount;
+            } else {
+              if (type === 'display') {
+                return "(None)";
+              } else {
+                return "";
+              }
+            }
+          }
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/list_partition.js]: `bSortable: false` for both tables

Comment (apmasell) [reply miso-web/src/main/webapp/scripts/list_partition.js]: Fixed.

**Comment by apmasell** [comment]
I have no strong feelings on the creation date. It's there for hysterical raisins.

**Pull request merged**